sapi/fpm/fpm/fpm_unix.c

```diff
@@ -88,8 +112,10 @@ bool fpm_unix_test_config(struct fpm_worker_pool_s *wp)
 {
 	struct fpm_worker_pool_config_s *config = wp->config;
 	return (
+		fpm_unix_check_listen_address(wp, config->listen_address, ZLOG_ERROR) &&
 		fpm_unix_check_passwd(wp, config->user, ZLOG_ERROR) &&
 		fpm_unix_check_group(wp, config->group, ZLOG_ERROR) &&
+        fpm_unix_check_listen_address(wp, config->listen_address, ZLOG_SYSERROR) &&
```

Comment:
why is this being done twice?

Reply:
You are right. I was adding it twice, one for ZLOG_ERROR and one for ZLOG_SYSERROR as this has been done twice for user and group as well, but I misread the lines. It was checking four different configuration values. So is this rather a ZLOG_ERROR or a ZLOG_SYSERROR? I tend to go for ZLOG_SYSERROR

Reply:
Yeah agreed ZLOG_SYSERROR is more consistent...

Reply:
I addressed this.